fiddler/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# CHANGELOG - fiddler
+
+## 1.0.0 / 2022-05-07
```

Comment:
@rajeevthefiddler, let's update the release date to `2022-10-20`. Our release pipeline is currently frozen for maintenance, but we can make another PR to change the date if the check is released on a different date than that.

Reply:
Done